cookbook/bundles/best_practices.rst

```diff
@@ -102,9 +101,8 @@ that automated tools can rely on:
 * ``README.md``: This file contains the basic description of the bundle and it
   usually shows some basic examples and links to its full documentation (it
   can use any of the markup formats supported by GitHub, such as ``README.rst``);
-* ``Resources/meta/LICENSE``: The full license for the code. The license file
-  can also be stored in the bundle's root directory to follow the generic
-  conventions about packages;
+* ``LICENSE``: The full license for the code. This license file can also be stored
+  in the bundle's ``Resources/meta/`` directory for backwards compatibility reasons;
```

Comment:
@OskarStark imo, "strongly encouraged" = we almost force you to choose MIT, but you may choose another license if you have very good reasons. "most usefull for your end-users" = you're free to choose another license, but be aware of license conflicts for your users.

And please note that MIT !== MIT-compatible (there are a lot more MIT-compatible licenses).

And yes, I agree that it's a good idea to link to http://choosealicense.com/